docs/example/one-page.md

```
@@ -6,8 +6,11 @@ title: vitepress-openapi
 
 <script setup lang="ts">
 import { useData } from 'vitepress'
+import ThemeConfiguration from '@dev/.vitepress/theme/components/ThemeConfiguration.vue'
 
 const { isDark } = useData()
 </script>
 
+<ThemeConfiguration />
```

Markdownlint

Element: ThemeConfiguration
Inline HTML

(MD033, no-inline-html)